src/js/modules/articles/components/RightRail.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { connect } from "react-redux";
+import injectSheet from "react-jss";
+import { Link } from "react-router-dom";
+
+import Byline from "./Byline";
+import { getArticlesWithContributors } from "../selectors";
+
+const NUMBER_OF_RAIL_ARTICLES = 5;
+
+const styles = {
+  RightRail: {
+    marginTop: "28px",
+  },
+  label: {
+    borderTop: "1px solid #000",
+    borderBottom: "1px solid #ddd",
+    color: "#000",
+    display: "block",
+    fontFamily: "Circular Std",
+    fontSize: "13px",
+    fontWeight: 300,
+    margin: "0 0 12px 0",
+    padding: "4px 0",
+    "&:hover": {
+      color: "#000",
+    },
+    "&:focus": {
+      color: "#000",
+    },
+  },
+  article: {
+    borderBottom: "solid 1px #ddd",
+    paddingBottom: "9px",
+    marginBottom: "7px",
+  },
+  bigTitle: {
+    color: "#000",
+    fontFamily: "Minion Pro",
+    fontSize: "18px",
+    fontWeight: "bold",
+    lineHeight: "1.22",
+    marginBottom: "7px",
+  },
+  smallTitle: {
+    color: "#000",
+    fontFamily: "Minion Pro",
+    fontSize: "15px",
+    lineHeight: "1.25",
+  },
+  sectionLabel: {
+    color: "#000",
+    display: "block",
+    fontFamily: "Minion Pro",
+    fontSize: "12px",
+    marginBottom: "6px",
+    textTransform: "uppercase",
+    "&:hover": { color: "#000", textDecoration: "none" },
+    "&:focus": { color: "#000", textDecoration: "none" },
+  },
+  figure: {
+    float: "right",
+    marginLeft: "5px",
+    width: "62px",
+    "& img": {
+      width: "100%",
+    },
+  },
+  Byline: {
+    color: "#888",
+    fontFamily: "Circular Std",
+    fontSize: "12px",
+    fontWeight: "300",
+    marginBottom: "3px",
+    "& p": {
+      margin: "0",
+      display: "inline",
+      "& a": {
+        color: "#888",
+        "&:hover": {
+          color: "#888",
+        },
+      },
+    },
+  },
+  Dateline: {
+    color: "#888",
+    fontFamily: "Circular Std",
+    fontSize: "12px",
+    fontWeight: "300",
+    margin: 0,
+    "& p": {
+      color: "#000",
+      margin: 0,
+      display: "inline",
+    },
+  },
+  "@media (max-width: 991px)": {
+    RightRail: {
+      paddingLeft: "1.5vw",
+    },
+  },
+};
+
+// inside a Col
+const RightRail = ({ classes, articles, sections, media }) => {
+  return (
+    <div className={classes.RightRail}>
+      <Link to="/recommended" className={classes.label}>
+        Recommended
+      </Link>
+      {Object.values(articles)
```

Comment:
Huh, why is this different from the component above? I like how you made a separate variable above, but I like the constant here. Do both in both